baybe/recommenders/base.py

```diff
@@ -55,40 +55,20 @@ class Recommender(ABC, RecommenderProtocol):
     """Allow to make recommendations that were measured previously.
     This only has an influence in discrete search spaces."""
 
-    def recommend(  # noqa: D102
+    def recommend(
         self,
         searchspace: SearchSpace,
         batch_size: int = 1,
         train_x: Optional[pd.DataFrame] = None,
         train_y: Optional[pd.DataFrame] = None,
     ) -> pd.DataFrame:
-        # See base class.
-
-        if searchspace.type == SearchSpaceType.DISCRETE:
-            # Select, recommend, and mark discrete candidates in one atomic step
-            return self._select_candidates_and_recommend(
-                searchspace.discrete,
-                batch_size,
-            )
-
-        if searchspace.type == SearchSpaceType.CONTINUOUS:
+        """See base class."""
```

Comment:
Please change this back. Using the exception results in the method inheriting the actual docstring of the parent class. This solution replaces it by `See base class` which is definitely inferior.

Reply:
changed

this is till a very bad way of fixing the issue, no one (eg external people) will understand or remember unless repeatedly doing it this way

Reply:
Nope, we actually mention this in our guidelines: 
![image](https://github.com/emdgroup/baybe/assets/84016733/8ba461ef-462c-430f-a8b0-20bc1a0c0299)